developers/dev-faq.md

```diff
@@ -0,0 +1,82 @@
+
+</details>
+
+<details>
+<summary>How are datatokens created, and how does the creator make money?</summary>
+
+[Datatokens](../developers/contracts/datatokens) represent access rights to a specific dataset or data service on the Ocean Protocol. Creators of datasets tokenize their data, creating these datatokens. Each datatoken is unique to a dataset.
+</details>
+
+<details>
+<summary>Where can I find information about the number of datatokens created and track their progress?</summary>
+
+To access this data, some technical expertise is required. You can find this information at the subgraph level. In the documentation, we provide a few examples of how to retrieve this data using JavaScript. Feel free to give it a shot by visiting this [page](../developers/subgraph/list-datatokens). If it doesn't meet your requirements, don't hesitate to reach out to us on Discord.
+</details>
+
+<details>
+<summary>How can collaborators and developers use Ocean technology to build their own data marketplaces?</summary>
+
+Ocean technology is open-source, community-funded, and freely available for use by anyone. 
+Anyone can fork various components from the Ocean [GitHub](https://github.com/oceanprotocol) repository and create their own. Additionally, Ocean empowers dApp owners with great flexibility and control over the fees they charge, allowing you to customize the fee structure to meet your specific needs and ensure your project's sustainability.
+</details>
+
+<details>
+<summary>Is there a trading platform or stock exchange that has successfully forked the Ocean marketplace codebase?</summary>
+
+Ocean technology is actively used by Daimler/Acentrik, deltaDAO/GAIA-X, and several other entities. You can find further details on the Ocean [ecosystem page](https://oceanprotocol.com/explore/ecosystem).
+
+</details>
+
+<details>
+<summary>What are the Ocean faucets and how can they be used?</summary>
+
+An Ocean faucet is a site to get (fake) OCEAN for use on a given testnet. There's an Ocean faucet for each testnet that Ocean is deployed to. The [networks](../discover/networks/) page have more information.
+</details>
+
+<details>
+<summary>Can I use the off-the-shelf CSS available in the repo?</summary>
+
+The marketplace name, logo, and typeface must be changed by the client. A slight modification would be enough for compliance. For more information consult the README file about [forking Ocean Market](https://github.com/oceanprotocol/market#-forking). 
+</details>
+
+<details>
+<summary>How can I convert tokens from the BEP20 network to the ERC20 network?</summary>
+
+Please follow this [tutorial](../discover/networks/bridges#binance-smart-chain-bsc-bridge) to bridge from/to binance smart chain. Please double-check the addresses and make sure you are using the right smart contracts.
```

Comment:
Change "binance smart chain" --> "BNB Smart Chain (nee Binance Smart Chain)"

Why: it's been renamed, and need to capitalize properly.